src/onefetch/git_utils.rs

```diff
@@ -74,7 +74,20 @@ pub fn get_number_of_commits(git_history: &[String]) -> String {
     number_of_commits.to_string()
 }
 
-pub fn get_packed_size(dir: &str) -> Result<String> {
+pub fn get_packed_size(repo_size: String, files_count: Option<u128>) -> Result<String> {
+    match files_count {
+        Some(files_count) => {
+            let res = repo_size + (" (") + &(files_count.to_string()) + (" files)");
```

Comment:
Instead of string addition, could you use the `format!` macro? I think it's more readable.

Reply:
I can change that!